### ISSUE TITLE
Next step for XVG integration?

### ISSUE THREAD

**Issue by CryptoKaku** [LedgerHQ/ledger-nano-s#50]
Please see https://twitter.com/vergecurrency/status/1060240916483465216 regarding XVG integration - it seems the XVG development team has been waiting several months for a response, thanks.

Would you please see if there is anything that can be done to push this forward?  I really don't have a use for the Ledger Nano S if it doesn't support XVG.

Thank you.

**Comment by cryptoborg11** [comment]
Did someone response?

**Comment by CryptoKaku** [comment]
No, no one from Ledger as of yet has responded to this Github issue.  XVG core team has been waiting for months!
